quickfixj-codegenerator/src/main/resources/org/quickfixj/codegenerator/Fields.xsl

```diff
@@ -87,6 +95,17 @@ public class <xsl:value-of select="@name"/> extends <xsl:call-template name="get
 		super(<xsl:value-of select="@number"/>, new <xsl:value-of select="$dataType"/>(data));
 	}
 	</xsl:if>
+
+    <xsl:choose>
+        <xsl:when test="$utcTimestampPrecision">
+            <xsl:if test="@type='UTCTIMESTAMP' or @type='UTCTIME' or @type='UTCTIMEONLY'">
+    @Override
+    protected UtcTimestampPrecision getDefaultUtcTimestampPrecision() {
+        return UtcTimestampPrecision.<xsl:value-of select="$utcTimestampPrecision"/>;
+    }
+            </xsl:if>
```

Comment:
Trivial. In the generated classes there is an empty line here. Not sure which xsl part generates this.

Reply:
Fixed in 95c1373e1968d968e3002751c931d501c3d9c0f3.